DuckDuckGo/SettingsDebugView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SettingsDebugView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import UIKit
+
+struct SettingsDebugView: View {
+    
+    @EnvironmentObject var viewModel: SettingsViewModel
+    
+    var body: some View {
+        Section(header: Text("Debug")) {
+            
+            SettingsCellView(label: "All debug options",
+                             action: { viewModel.presentLegacyView(.debug) },
```

Comment:
I do not want to be fussy but maybe there was nothing wrong with the old header-less "Debug menu".